<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="activationOrder">
       <enum>QMdiArea::StackingOrder</enum>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabsMovable">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_Find"/>
    <addaction name="action_Replace"/>
    <addaction name="separator"/>
    <addaction name="action_Codec"/>
    <addaction name="action_Transcode"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
    <addaction name="action_Close"/>
    <addaction name="action_CloseAll"/>
    <addaction name="separator"/>
    <addaction name="action_Next"/>
    <addaction name="action_Previous"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
    <addaction name="action_WrapLine"/>
    <addaction name="separator"/>
    <addaction name="action_ZoomIn"/>
    <addaction name="action_ZoomOut"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_V"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="action_Save"/>
   <addaction name="action_SaveAs"/>
   <addaction name="separator"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="separator"/>
   <addaction name="action_Find"/>
   <addaction name="action_Replace"/>
   <addaction name="separator"/>
   <addaction name="action_ZoomIn"/>
   <addaction name="action_ZoomOut"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/new.png</normaloff>:/myImage/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建文件(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/open.png</normaloff>:/myImage/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开文件(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/save.png</normaloff>:/myImage/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/saveAs.png</normaloff>:/myImage/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>另存为(A)</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/exit.png</normaloff>:/myImage/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>退出(X)</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/undo.png</normaloff>:/myImage/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="toolTip">
    <string>撤销(U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/redo.png</normaloff>:/myImage/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>恢复(R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/cut.png</normaloff>:/myImage/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>剪切(T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/copy.png</normaloff>:/myImage/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>复制(C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/paste.png</normaloff>:/myImage/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>粘贴(P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>关闭当前(&amp;Q)</string>
   </property>
   <property name="toolTip">
    <string>关闭当前(Q)</string>
   </property>
  </action>
  <action name="action_CloseAll">
   <property name="text">
    <string>关闭所有(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关闭所有(A)</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/next.png</normaloff>:/myImage/images/next.png</iconset>
   </property>
   <property name="text">
    <string>下一个(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>下一个(X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="action_Previous">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/previous.png</normaloff>:/myImage/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>前一个(&amp;V)</string>
   </property>
   <property name="toolTip">
    <string>前一个(V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Backtab</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/about.png</normaloff>:/myImage/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于(A)</string>
   </property>
  </action>
  <action name="action_WrapLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
   <property name="toolTip">
    <string>自动换行(W)</string>
   </property>
  </action>
  <action name="action_ZoomIn">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/zoomIn.png</normaloff>:/myImage/images/zoomIn.png</iconset>
   </property>
   <property name="text">
    <string>放大(&amp;I)</string>
   </property>
   <property name="toolTip">
    <string>放大(I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_ZoomOut">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/zoomOut.png</normaloff>:/myImage/images/zoomOut.png</iconset>
   </property>
   <property name="text">
    <string>缩小(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>缩小(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/qt.png</normaloff>:/myImage/images/qt.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt(&amp;Q)</string>
   </property>
   <property name="toolTip">
    <string>关于Qt(Q)</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/find.png</normaloff>:/myImage/images/find.png</iconset>
   </property>
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>查找(F)</string>
   </property>
  </action>
  <action name="action_Replace">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/replace.png</normaloff>:/myImage/images/replace.png</iconset>
   </property>
   <property name="text">
    <string>替换(&amp;L)</string>
   </property>
   <property name="toolTip">
    <string>替换(L)</string>
   </property>
  </action>
  <action name="action_Codec">
   <property name="text">
    <string>选择编码(&amp;D)</string>
   </property>
   <property name="toolTip">
    <string>选择编码(D)</string>
   </property>
  </action>
  <action name="action_Transcode">
   <property name="text">
    <string>转换编码(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>转换编码(A)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myimage.qrc"/>
 </resources>
 <connections/>
</ui>
